bin/rucio-admin

```diff
@@ -916,6 +916,11 @@ def declare_bad_file_replicas(args):
             for rse in non_declared:
                 for pfn in non_declared[rse]:
                     print('%s : PFN %s cannot be declared.' % (rse, pfn))
+
+    successfully_declared_pfns = [pfn for pfn in bad_files_pfns if pfn not in [not_declared_pfn.split(' ')[0] for entry in list(non_declared.values()) for not_declared_pfn in entry]]
```

Comment:
Whoops, forgot to commit my previous review.

Wouldn't it make more sense to do this the other way round? Basically say "Everything went fine" and if not, display only those that didn't work? So it won't spam the screen for naught?

Reply:
ah yes, makes more sense